modules/apps/journal/journal-content-web/src/main/java/com/liferay/journal/content/web/internal/display/context/JournalContentDisplayContext.java

```diff
@@ -742,6 +804,27 @@ public void incrementViewCounter() throws PortalException {
 	}
 
 	public boolean isDefaultTemplate() {
+		if (FeatureFlagManagerUtil.isEnabled(
+				_themeDisplay.getCompanyId(), "LPD-27566")) {
+
+			String ddmTemplateExternalReferenceCode = ParamUtil.getString(
+				_portletRequest, "ddmTemplateExternalReferenceCode");
+
+			if (Validator.isNotNull(ddmTemplateExternalReferenceCode)) {
+				return false;
+			}
+
+			ddmTemplateExternalReferenceCode =
+				_journalContentPortletInstanceConfiguration.
+					ddmTemplateExternalReferenceCode();
+
+			if (Validator.isNotNull(ddmTemplateExternalReferenceCode)) {
+				return false;
+			}
+
+			return true;
+		}
+
 		String ddmTemplateKey = ParamUtil.getString(
```

Comment:
I know this is old code, but the same as before, why not fallback from _journalContentPortletInstanceConfiguration.ddmTemplateKey()

Reply:
Same as above.